package.json

```
@@ -5,6 +5,7 @@
     "build": "next build",
     "postbuild": "next-sitemap && ts-node etc/create-sitemap-index.ts",
     "dev": "next",
+    "analyze": "ANALYZE=true yarn build",
```

```suggestion
    "analyze": "cross-env ANALYZE=true yarn build",
```

This will fail to run on Windows. Please use `cross-env` for a cross-platform solution for setting environment variables.

---

Added